src/components/pages/document.js

```diff
@@ -277,33 +291,35 @@ function Document({ initialState, PageTemplate }) {
   }, [doc, historySelectedDocumentRevision, view, pendingTemplateSectionKeys, t]);
 
   useEffect(() => {
-    if (preSetView === VIEW.history) {
+    if (preSetView === VIEW.history && !initialDocumentRevisionsFetched) {
```

Comment:
I guess it would be simpler and clearer to just early return when `initialDocumentRevisionsFetched`, to not repeat it in these 2 scenarios 🤔

Reply:
..One would, say, if both conditions included the same flag 😮‍💨 But they do not 🤦‍♀️

Reply:
It's not repeated, the first condition is `!initialDocumentRevisionsFetched`, the second one is `!initialDocumentCommentsFetched` :thinking:

Reply:
Ah, sorry, didn't reload and missed your second message :-)